src/extension.ts

```diff
@@ -127,7 +127,8 @@ export async function activate(context): Promise<extensionapi.ExtensionAPI> {
         registerCommand('extension.vsKubernetesLoad', loadKubernetes),
         registerCommand('extension.vsKubernetesGet', getKubernetes),
         registerCommand('extension.vsKubernetesRun', runKubernetes),
-        registerCommand('extension.vsKubernetesLogs', logsKubernetes),
+        registerCommand('extension.vsKubernetesShowLogs', logsKubernetes),
+        registerCommand('extension.vsKubernetesFollowLogs', (obj) => {logsKubernetes(obj, true);}),
```

Comment:
hmm - let's try to give `obj` a better name here.

and also the type - looks like it's `explorer.ResourceNode`

Reply:
added meaningful name and type